front/app/containers/Admin/communityMonitor/components/LiveMonitor/components/FormResults.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import { useParams } from 'react-router-dom';
+
+import useProjectById from 'api/projects/useProjectById';
+import useFormResults from 'api/survey_results/useSurveyResults';
+
+import SentimentQuestion from 'components/admin/FormResults/FormResultsQuestion/SentimentQuestion';
+
+import { useIntl } from 'utils/cl-intl';
+
+import messages from '../messages';
+
+type Props = {
+  projectId?: string;
+  phaseId?: string;
+};
+
+const FormResults = (props: Props) => {
+  const { projectId: projectIdParam, phaseId: phaseIdParam } = useParams() as {
+    projectId: string;
+    phaseId: string;
+  };
+
+  const projectId = props.projectId || projectIdParam;
+  const phaseId = props.phaseId || phaseIdParam;
+
+  const { formatMessage } = useIntl();
+  const { data: project } = useProjectById(projectId);
+
+  const { data: formResults } = useFormResults({
+    phaseId,
+    filterLogicIds: [],
+  });
+
+  if (!formResults || !project) {
+    return null;
+  }
+
+  const { totalSubmissions, results } = formResults.data.attributes;
+
+  return (
+    <Box width="100%">
+      {totalSubmissions === 0 && (
+        <Box width="100%">
+          <Text variant="bodyM" color="textSecondary">
+            {formatMessage(messages.noSurveyResponses)}
+          </Text>
+        </Box>
+      )}
+      <Box>
+        {totalSubmissions > 0 &&
+          results.map((result, index) => {
+            if (result.inputType === 'sentiment_linear_scale') {
+              return <SentimentQuestion key={index} result={result} mb="8px" />;
+            }
+
+            return null;
+          })}
+      </Box>
```

Comment:
Minor: We could filter out what we want and then render. Also using a ternary since both options can't be right
```suggestion
      {totalSubmissions === 0 ? (
        <Box width="100%">
          <Text variant="bodyM" color="textSecondary">
            {formatMessage(messages.noSurveyResponses)}
          </Text>
        </Box>
      ) : (
      <Box>
        {results
          .filter((result) => result.inputType === 'sentiment_linear_scale')
          .map((result, index) => (
            <SentimentQuestion key={index} result={result} mb="8px" />
          ))}
      </Box>
      )}
```

Reply:
Good suggestion! :+1: Thanks!!